src/views/Login.vue

```diff
@@ -45,15 +50,36 @@
 </template>
 
 <script>
+import BaseButton from '@/components/BaseButton.vue';
+import authService from '@/api/authService';
+import { get } from 'lodash';
```

Comment:
Import just the get method
import get from 'loadash/get'

Reply:
+ nu trebuie importat BaseButton, pentru ca e inclus in pluginul de globalComponents si se importa global